### ISSUE TITLE
Make sure "specialized" definitions reference their "primitive" concepts

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#3706]
I noticed during #3702 that the definition of "position vector" doesn't reference "position" at all! Regardless of whether this definition is meant to convey something different (i.e., if the fix is renaming it vs. redefining it), an investigation of how we "build up" concepts should probably be done as part of the work from that meeting. Thoughts?

https://github.com/JacquesCarette/Drasil/blob/1ef469d9eefe1dd6b9f8d0ddac665951445d2a8d/code/drasil-data/lib/Data/Drasil/Concepts/Physics.hs#L137-L141

**Comment by smiths** [comment]
Thank you for creating this issue @samm82.  Yes, we need to go through all of our examples and strengthen the concepts.

**Comment by Xinlu-Y** [comment]
It seems that `position` is being used in the key model definitions and equations. Could this be a result of inconsistent naming, or should `positionVec` replace `position` in this context? Additionally, how should I correctly build up `positionVec` to reference `position`?
https://github.com/JacquesCarette/Drasil/blob/8624ac8d9bb18083f1df9265ff9b311b5df737cc/code/drasil-example/projectile/lib/Drasil/Projectile/GenDefs.hs#L97-L112
![Image](https://github.com/user-attachments/assets/e8db1abd-233f-4ee9-9d78-603e2544ebe7)

**Comment by JacquesCarette** [comment]
You are correct that there seems to be a confusion in this example between `position` and `positionVec`. Your instincts are correct that `position` should not be used at all, and `positionVec` should be instead. Is that clear enough?

**Comment by JacquesCarette** [comment]
It does make sense to define the 'concept' of `position`; `positionVec` should, in theory, refer to that. But our current arrangement of Chunks may not really support that at the moment. It's part of the big project of re-arranging chunks more 'semantically'.
